src/SDKs/DataMigration/DataMigration-ReRecordAfterCRUpdate.playlist

```diff
@@ -1 +0,0 @@
-<Playlist Version="1.0"><Add Test="DataMigration.Tests.ScenarioTests.CRUDServiceTests.DeleteResourceSucceeds" /></Playlist>
```

Comment:
Was this intentional?

Reply:
@shahabhijeet sent out an email about tests with this file will have to be re-recorded.
I have re-recorded all the tests and so I assumed it should be safe to remove this file.

Reply:
@shahabhijeet would it be safe to remove this file?